src/plugins/intel_gpu/src/graph/compilation_context.cpp

```diff
@@ -13,7 +13,7 @@ namespace cldnn {
 class CompilationContext : public ICompilationContext {
 public:
     CompilationContext(ov::threading::IStreamsExecutor::Config task_executor_config) : _task_executor_config(task_executor_config) {
-        _task_executor_config._streams = 4;
+        _task_executor_config._streams = 1;
```

Comment:
Could you please make a change so that this value is set from the caller? 
i.e., set _streams in task_executor_config from the caller side.

Reply:
Okay, I'll update it. Thank you.